core/src/state/mod.rs

```diff
@@ -538,7 +549,7 @@ impl<B: Backend> State<B> {
             .readonly(self.db.as_hashstore(), &self.root)
             .expect(SEC_TRIE_DB_UNWRAP_STR);
         let maybe_acc = db.get_with(address, AionVMAccount::from_rlp)?;
-        let r = maybe_acc.as_ref().map_or(Ok(vec![]), |a| {
+        let r = maybe_acc.as_ref().map_or(Ok(Some(vec![])), |a| {
```

Comment:
this default vec![] should be None instead. Although maybe_acc will not be None during avm getting storage